go/vt/vtgate/planbuilder/operators/vindex.go

```diff
@@ -67,70 +67,70 @@ func (v *Vindex) Clone(inputs []Operator) Operator {
 
 var _ PhysicalOperator = (*Vindex)(nil)
 
-func (v *Vindex) PushOutputColumns(columns []*sqlparser.ColName) ([]int, error) {
-	idxs := make([]int, len(columns))
-outer:
-	for i, newCol := range columns {
-		for j, existingCol := range v.Columns {
-			if sqlparser.EqualsExpr(newCol, existingCol) {
-				idxs[i] = j
-				continue outer
-			}
+func (v *Vindex) AddColumn(_ *plancontext.PlanningContext, expr sqlparser.Expr) (int, error) {
+	// TODO: unify with table
+	col, ok := expr.(*sqlparser.ColName)
```

Comment:
Do you want to handle this TODO in this PR or a follow-up for later? I am fine either way